index.html

```diff
@@ -83,37 +84,181 @@ <h5 class="card__title"><img src="./img/svg/suitcase.svg" class="card-icon"
+                        <div>
+                            <a href="#!" class="client__name">Roger Brown</a>
+                            <h6 class="client__job">Project Manager</h6>
+                        </div>
+                    </div>
+                </div>   
+            </div>
     </section>
 
-</body>
+    <section class="our-team">
+        <div class="container">
+            <div class="our-team__content">
+                <h2 class="content__general-title">Meet Our Team</h2>
+                <p class="content__text">Problems trying to resolve the conflict between <br> the two major realms of
+                    Classical physics: Newtonian mechanics<p>
+            </div>
 
+            <div class="grid-container">
+                <div class="grid-card" id="card-staff">
+                    <div class="staff-info">
+                        <img src="./img/staff-ashley-fletcher.png" alt="Ashley Fletcher photo" class="staff__photo">
+                        <div>
+                            <div class="staff__content">
+                                <h5 class="staff__name">Ashley Fletcher</h5>
+                                <h6 class="staff__job">CEO</h6>
+                                <div class="staff__social-media">
+                                    <img src="./img/svg/facebook-icon.svg" alt="Facebook icon" class="social-media">
+                                    <img src="./img/svg/instagram-icon.svg" alt="Instagram icon icon"
+                                        class="social-media">
+                                    <img src="./img/svg/twitter-icon.svg" alt="Twitter icon" class="social-media">
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+
+                <div class="grid-card" id="card-staff">
+                    <div class="staff-info">
+                        <img src="./img/staff-rodney-stratton.png" alt="Rodney Stratton photo" class="staff__photo">
+                        <div>
+                            <div class="staff__content">
+                                <h5 class="staff__name">Rodney Stratton</h5>
+                                <h6 class="staff__job">Consultant</h6>
+                                <div class="staff__social-media">
+                                    <img src="./img/svg/facebook-icon.svg" alt="Facebook icon" class="social-media">
+                                    <img src="./img/svg/instagram-icon.svg" alt="Instagram icon icon"
+                                        class="social-media">
+                                    <img src="./img/svg/twitter-icon.svg" alt="Twitter icon" class="social-media">
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+
+                <div class="grid-card" id="card-staff">
```

Comment:
id should be unique.
You use `id="card-staff"` several times, which is forbidden

Reply:
it's not only about card-staff. Check all your ids

Reply:
done!

Reply:
@IvanCeban thanks a lot for the detailed explanation! Now it`s clear